redbeams/src/main/java/com/sequenceiq/redbeams/configuration/DatabaseServerSslCertificateConfig.java

```diff
@@ -33,6 +33,14 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DatabaseServerSslCertificateConfig.class);
 
+    private static final Pattern CLOUD_PLATFORM_AND_REGION_PATTERN = Pattern.compile("([a-zA-Z]+)(?:\\.([a-zA-Z0-9-]+))?");
```

Comment:
I would prefer to have the two supported cloud platform in the pattern, it would make it more clear.

Reply:
Good point. The regex has been fixed (explicitly mentioning the valid cloud platform pattern `aws|azure`) and 2 new UT cases have been added accordingly.